src/api/app/views/webui2/webui/request/_show_side_links.html.haml

```diff
@@ -0,0 +1,45 @@
+%ul.list-unstyled
+  %li
+    %i.fas.fa-user.text-success
+    Created
+    by #{user_with_realname_and_icon(request_creator, no_icon: true, short: true)} #{fuzzy_time(request_created_at)}
+  %li
+    %i.fa.fa-info-circle.text-info{ title: 'Info' }
+    In
+    state #{link_to(request_state, { anchor: 'request_history' }, style: "color: #{request_state_color(request_state)};")}
+  - unless package_maintainers.empty?
+    %li
+      %i.fas.fa-eye.text-info{ title: 'Look into' }
+      = pluralize(package_maintainers.size, 'package maintainer')
+      (#{link_to('show who', '#', class: 'show_dialog', data: { target: '#maintainers_dialog' })})
+  - if superseding
+    %li
+      %i.fas.fa-plus-circle.text-primary
+      Supersedes
+      - superseding.each do |supersed|
+        = link_to supersed['number'], number: supersed['number']
+  - if superseded_by
+    %li
+      %i.fa.fa-exclamation-circle.text-info{ title: 'Alert' }
+      Superseded
+      by #{link_to superseded_by, number: superseded_by}
+  - if accept_at
+    %li
+      %i.fa.fa-exclamation-circle.text-info{ title: 'Alert' }
```

Comment:
The title doesn't bring value, let's remove it.

The icon `fa-exclamation-circle` should be paired with `text-danger`, not with `text-info`.